### PR TITLE
Provide GitHub templates for new issues and PRs #85

### PR DESCRIPTION
**Pull request by djelinek**
Signed-off-by: Dominik Jelinek <djelinek@redhat.com>

### CODE REVIEW

**Comment by djelinek** [comment]
Related to #85

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+> Define issue title **short** and **most specific** to the problem.
+
+> Tag each new issue with relevant **LABELS** (*Bug*, *Improvement*, etc.).
+
+## **BUG** report
+
+### Environment
+
+- Apache Camel Language Server release version date: e.g. "**1.0.0-SNAPSHOT**, **Jun-04**" ?
```
Comment (apupier) [inline .github/ISSUE_TEMPLATE.md]: I think users won't know this version. They will know only the version of VS Code extension

Comment (djelinek) [reply .github/ISSUE_TEMPLATE.md]: so better to use something like: 

- Language Support for Apache Camel version: e.g. "**v0.0.13**" ?

Comment (apupier) [reply .github/ISSUE_TEMPLATE.md]: yes

**Comment by djelinek** [comment]
@apupier I forgot to this issue, could we merge it now? or do we want to make some additional modifications before merge?

**Pull request merged**